### PR TITLE
CNRS benchmark Phase 1 input files

### PR DESCRIPTION
**Pull request by smpark7**
This PR adds Moltres input files to run Phase 1 of the CNRS benchmark. The CNRS benchmark is a multiphysics benchmark for MSR simulation software. More details about the CNRS benchmark can be found in "Results from a multi-physics numerical benchmark for codes dedicated to molten salt fast reactors" ([DOI](https://doi.org/10.1016/j.anucene.2020.107428))

Changes from this PR:
- Moved Phase 0 input files to its own subdirectory `phase-0/`
- Moltres input files for Steps 1.1, 1.2, 1.3, & 1.4 of Phase 1
- Dedicated readmes in `phase-0/` and `phase-1/` describing the relevant input and output files for each Step, and instructions for running the input files on a 40-by-40 mesh on personal computers which do not have enough computing power and/or RAM to run on the original 200-by-200 mesh
- The readmes also include reactivity/k_eff results for both 200-by-200 and 40-by-40 mesh options

### CODE REVIEW

File: problems/2021-cnrs-benchmark/phase-1/full-coupling.i [hunk 1]
```diff
@@ -0,0 +1,314 @@
+density = .002  # kg cm-3
+cp = 3075       # J kg-1 K-1, 6.15 / 2.0e-3
+k = .005        # W cm-1 K-1
+gamma = 1       # W cm-3 K-1, Volumetric heat transfer coefficient
+viscosity = .5  # dynamic viscosity
+alpha = 1       # SUPG stabilization parameter
+t_alpha = 2e-4  # K-1, Thermal expansion coefficient
```
Comment (robfairh) [inline problems/2021-cnrs-benchmark/phase-1/full-coupling.i:7]: where did you get this number from?

**Review by robfairh — APPROVED** [review]
Great job!

File: problems/2021-cnrs-benchmark/phase-1/full-coupling.i (hunk 1 above)
Comment (robfairh) [reply problems/2021-cnrs-benchmark/phase-1/full-coupling.i:7]: I am curious also about how the benchmark exercise is defined.
Is it defined to be solved with the Boussinesq approximation or to solve the density equation as well?
How does CNRS solve it?
@smpark7

Comment (smpark7) [reply problems/2021-cnrs-benchmark/phase-1/full-coupling.i:7]: Hey @robfairh! `gamma`, `viscosity`, and `t_alpha` are all directly from the CNRS benchmark journal article linked above. The authors picked a 2m-by-2m 2D domain for the geometry. All co-authors used incompressible Navier-Stokes with the Boussinesq approximation for their own multiphysics simulation tools.

**Pull request merged**